heartbeat/monitors/active/icmp/loop.go

```diff
@@ -107,6 +121,17 @@ func newICMPLoop() (*icmpLoop, error) {
 		go l.runICMPRecv(conn6, protocolIPv6ICMP)
 	}
 
+	if conn4 == nil && conn6 == nil {
+		if unprivilegedPossible {
+			path, _ := os.Executable()
+			logp.Info("You dont have root permission to run ping. You can run without root by setting cap_net_raw:")
+			logp.Info("sudo setcap cap_net_raw+eip %s", path)
+			logp.Info("Your system allows the use of unprivileged ping by setting net.ipv4.ping_group_range")
+			logp.Info("sysctl -w net.ipv4.ping_group_range='<min-uid> <max-uid>' ")
```

Comment:
Better use a multiline string instead of multiple calls to `logp.Info`. Beats has a many go-routines and there is a high chance of the log message being intermingled with other messages if not done so. Beats also has a json structured logger, where multiline is not that natural.

Although it's nice to have a more detailed log message, this message is very linux specific, no?

Reply:
Yes, indeed it is... :) changing...